cmd/report/main.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/k8sbykeshed/k8s-service-validator/tools"
+	"github.com/savaki/jq"
+	"go.uber.org/zap"
+)
+
+type Workflow struct {
+	name               string
+	status             string
+	id                 string
+	lastRunID          string
+	lastJobID          string
+	lastJobCompletedAt string
+	passedTests        map[string]bool
+	failedTests        map[string]bool
+}
+
+var wfmap map[string]*Workflow
+
+const (
+	cmd          = "/usr/local/bin/gh"
+	keyPassInLog = "PASS tests"
+	keyFailInLog = "FAIL: tests"
+)
+
+func main() {
+	// TODO gh auth login
```

Comment:
can you create an issue for it?

Reply:
https://github.com/K8sbykeshed/k8s-service-validator/issues/111